src/App.tsx

```diff
@@ -58,58 +58,60 @@ function App() {
-          vertical
-        >
-          <NavBar theme={imageTheme} />
+        <AuthProvider>
           <Flex
+            style={{ width: '100vw', maxWidth: '1024px' }}
             align='center'
-            justify='space-evenly'
-            className='flex-auto w-full'
+            justify='space-between'
+            className='h-full w-full '
             vertical
           >
-            {showApp ? <PhotoQueue images={images} /> : dailyGameReminder}
-          </Flex>
-          <div className='text-center m-8'>
-            <p>
-              Made by{' '}
-              <a
-                href='https://faisal-fawad.github.io'
-                onClick={() => posthog.capture('faisal_link_click')}
-                target='_blank'
-              >
-                Faisal
-              </a>
-              ,{' '}
-              <a
-                href='https://nathanprobert.ca'
-                onClick={() => posthog.capture('nate_link_click')}
-                target='_blank'
-              >
-                Nate
-              </a>
-              , and{' '}
+            <NavBar theme={imageTheme} />
+            <Flex
+              align='center'
+              justify='space-evenly'
+              className='flex-auto w-full'
+              vertical
+            >
+              {showApp ? <PhotoQueue images={images} /> : dailyGameReminder}
+            </Flex>
+            <div className='text-center m-8'>
+              <p>
+                Made by{' '}
+                <a
+                  href='https://faisal-fawad.github.io'
+                  onClick={() => posthog.capture('faisal_link_click')}
+                  target='_blank'
+                >
+                  Faisal
+                </a>
+                ,{' '}
+                <a
+                  href='https://nathanprobert.ca'
+                  onClick={() => posthog.capture('nate_link_click')}
+                  target='_blank'
+                >
+                  Nate
+                </a>
+                , and{' '}
+                <a
+                  href='https://zachlegesse.ca'
+                  onClick={() => posthog.capture('zach_link_click')}
+                  target='_blank'
+                >
+                  Zach
+                </a>
+              </p>
               <a
-                href='https://zachlegesse.ca'
-                onClick={() => posthog.capture('zach_link_click')}
+                className='text-slate-500'
+                href='https://pexels.com'
                 target='_blank'
               >
-                Zach
+                Real photos sourced at Pexels.com
               </a>
-            </p>
-            <a
-              className='text-slate-500'
-              href='https://pexels.com'
-              target='_blank'
-            >
-              Real photos sourced at Pexels.com
-            </a>
-          </div>
-        </Flex>
+            </div>
+          </Flex>
+        </AuthProvider>
```

Comment:
The only thing I did in this file is wrap all of the tags with the `AuthProvider`

Reply:
This file needs to be cleaned up... oh well